hashing/te-tag-query-python/all_in_one_lib/signal_type/url.py

```diff
@@ -0,0 +1,24 @@
+##!/usr/bin/env python
+
+"""
+Wrapper around the URL content type.
+"""
+
+import typing as t
+
+from ..descriptor import SimpleDescriptorRollup, ThreatDescriptor
+from . import base
+
+
+class URLSignal(base.SimpleSignalType, base.StrMatcher):
+
+    INDICATOR_TYPE = "RAW_URI"
```

Comment:
Add note that this also includes URI but that my helper only handles one at a time